src/System.Management.Automation/utils/Verbs.cs

```diff
@@ -592,6 +592,1018 @@ public static class VerbsOther
+        /// Removes a resource from an indicated location
+        /// </summary>
+        public const string Uninstall = "Removes a resource from an indicated location";
+
+        /// <summary>
+        /// Releases a resource that was locked
+        /// </summary>
+        public const string Unlock = "Releases a resource that was locked";
+
+        /// <summary>
+        /// Removes safeguards from a resource that were added to prevent it from attack or loss
+        /// </summary>
+        public const string Unprotect = "Removes safeguards from a resource that were added to prevent it from attack or loss";
+
+        /// <summary>
+        /// Makes a resource unavailable to others
+        /// </summary>
+        public const string Unpublish = "Makes a resource unavailable to others";
+
+        /// <summary>
+        /// Removes the entry for a resource from a repository
+        /// </summary>
+        public const string Unregister = "Removes the entry for a resource from a repository";
+
+        /// <summary>
+        /// Brings a resource up-to-date to maintain its state, accuracy, conformance, or compliance
+        /// </summary>
+        public const string Update = "Brings a resource up-to-date to maintain its state, accuracy, conformance, or compliance";
+
+        /// <summary>
+        /// Uses or includes a resource to do something
+        /// </summary>
+        public const string Use = "Uses or includes a resource to do something";
+
+        /// <summary>
+        /// Pauses an operation until a specified event occurs
+        /// </summary>
+        public const string Wait = "Pauses an operation until a specified event occurs";
+
+        /// <summary>
+        /// Continually inspects or monitors a resource for changes
+        /// </summary>
+        public const string Watch = "Continually inspects or monitors a resource for changes";
+
+        /// <summary>
+        /// Adds information to a target
+        /// </summary>
+        public const string Write = "Adds information to a target";
+    }//VerbDescriptions
+
+    /// <summary>
+    /// Verb Alias prefixes.
+    /// </summary>
+    public static class VerbAliasPrefixes
+    {
+        /// <summary>
+        /// "Add" verb alias prefix
+        /// </summary>
+        public const string Add = "a";
+
+        /// <summary>
+        /// "Approve" verb alias prefix
+        /// </summary>
+        public const string Approve = "ap";
+
+        /// <summary>
+        /// "Assert" verb alias prefix
+        /// </summary>
+        public const string Assert = "as";
+
+        /// <summary>
+        /// "Backup" verb alias prefix
+        /// </summary>
+        public const string Backup = "ba";
+
+        /// <summary>
+        /// "Block" verb alias prefix
+        /// </summary>
+        public const string Block = "bl";
+
+        /// <summary>
+        /// "Build" verb alias prefix
+        /// </summary>
+        public const string Build = "";
```

Comment:
@HemantMahawar should probably be defining a prefix for the verbs which have none: `Build`, `Deploy`, `Optimize`, and `Resize`.

Reply:
- Build --> bd
- Deploy --> dp
- Optimize --> om
- Resize --> rs

/cc @joeyaiello